docs/zh/latest/plugins/ua-restriction.md

```diff
@@ -1,5 +1,11 @@
 ---
 title: ua-restriction
+keywords:
+  - APISIX
+  - API Gateway
+  - UA restriction
+  - ua-restriction
+description: 本文介绍了 Apache APISIX ua-restriction 插件的基本信息及使用方法。
```

Comment:
https://www.eteste.com/

Reply:
After looking through the docs, I noticed that

- Almost all of the plugin docs donot meet the 130-150 requirement in the `description` section
- No discussion or guide on how to write the `description` were found

So, if this is the result of a recent community discussion, I'd like to know the necessity and benefits of doing this?

We know that the `keywords` and `description` sections before each document are not displayed on the doc-site, they are related to SEO optimization and are for the crawler bots.

If we specify 130-150, then the `description` is quite detailed. What are the actual impacts of a detailed description for crawler bots and SEO? Is there any theoretical support? Positive or negative? Any trade-off?

And what about the `description` section in the body of the document? Should the content of both sections be exactly the same? If they are different, it would be a bit weird.

These are my concerns. Thanks in advance for your reply~

Reply:
At present, through ahrefs, many documents and blogs on APISIX website do not meet SEO standards, so we hope to change this situation. At present, some documents are already doing this. If you have other questions, we can chat in detail through IM.